src/components/Upload/CommonFields/index.jsx

```diff
@@ -65,23 +65,23 @@ CommonFields.propTypes = {
     nomos: PropTypes.bool.isRequired,
     ojo: PropTypes.bool.isRequired,
     package: PropTypes.bool.isRequired,
-  }).isRequired,
+  }),
   decider: PropTypes.shape({
     nomosMonk: PropTypes.bool.isRequired,
     bulkReused: PropTypes.bool.isRequired,
     newScanner: PropTypes.bool.isRequired,
     ojoDecider: PropTypes.bool.isRequired,
-  }).isRequired,
+  }),
   reuse: PropTypes.shape({
     reuseUpload: PropTypes.number.isRequired,
     reuseGroup: PropTypes.string.isRequired,
     reuseMain: PropTypes.bool.isRequired,
     reuseEnhanced: PropTypes.bool.isRequired,
     reuseReport: PropTypes.bool.isRequired,
     reuseCopyright: PropTypes.bool.isRequired,
-  }).isRequired,
-  handleChange: PropTypes.func.isRequired,
-  handleScanChange: PropTypes.func.isRequired,
+  }),
```

Comment:
Please explain the reason to remove `isRequired` from the fields.

Reply:
This is a common fields component, i.e., This is a single component which can control how and when to render different common fields. We do not need every component to be rendered every time within this component. Let's say that we only want the AccessLevel Component to be rendered so we should pass only the accessLevel flag prop that would ensure that only the access level component s rendered. Marking all the props as required will throw error in the console if even a single prop is not being passed through the component.

Reply:
Ah, understood. Thanks for explaining it.